src/components/birthday.scss

```diff
@@ -0,0 +1,45 @@
+$blue: #0747a6;
+
+.birthday {
+  margin: 20px 0;
+}
+
+.birthdayLabel {
+  display: block;
+  font-weight: bold;
+  margin: 5px 0;
+}
+
+.birthdayInput {
+  width: 100%;
+  box-sizing: border-box;
+  padding: 0.5em;
+  border: 1px solid blue;
```

Comment:
Do you use here the standard css blue color or your variable?
If variable you must write
 border: 1px solid $blue;
 
 If you use standard blue color so you do not need variable at line 1...